cli/compiler/AbstractCompiler.js

```diff
@@ -0,0 +1,62 @@
+import childProcess from 'child_process';
+import path from 'path';
+
+class Compiler {
```

Comment:
No big deal but may as well make the classname match the filename: `AssetCompiler`